src/main/java/org/sunbird/trending/controller/TrendingController.java

```diff
@@ -21,12 +22,12 @@ public ResponseEntity<Map<String, Object>> trendingSearch(
         return new ResponseEntity<>(compositeSearchRes, HttpStatus.OK);
     }
 
-    @PostMapping("admin/trending/content/search")
-    public ResponseEntity<Map<String, Object>> trendingContentSearch(
+    @PostMapping("/microsite/topFeatured/content/search")
```

Comment:
derive generic url as similar like content/context/search

Reply:
As discussed we are using , "content/aggregation/search"